### PR TITLE
fix(components): various fixes for banner, timestamp, and content dl

### PR DESCRIPTION
**Pull request by andrew-ronaldson**
Closes https://github.com/patternfly/patternfly/issues/6907, https://github.com/patternfly/patternfly/issues/6729 and https://github.com/patternfly/patternfly/issues/6871

### CODE REVIEW

File: src/patternfly/components/Content/content.scss [hunk 1]
```diff
@@ -327,7 +327,7 @@
     grid-template-columns: 1fr;
 
     @media screen and (min-width: $pf-v6-global--breakpoint--sm) {
-      grid-template: auto / auto 1fr;
+      grid-template: auto / 12ch 1fr;
       grid-row-gap: var(--#{$content}--dl--RowGap);
       grid-column-gap: var(--#{$content}--dl--ColumnGap);
```
Comment (mcoker) [inline src/patternfly/components/Content/content.scss:332]: The column gap for `<dl>`'s is 2xl and the row gap is md. Do we want to update that? Some thoughts:
* In figma, the description list default column gap is sm, and row gap is lg. The compact description list column gap is sm, and row gap is md
* Currently our description list component's column and row gaps match what's in figma
* We have gap spacer tokens here - https://github.com/patternfly/patternfly/blob/bdd2cc6ea5d1d66773e74935441880783c121df1/src/patternfly/base/tokens/tokens-default.scss#L467-L474
  * For the default `<dl>` we could use `gap--text-to-element` and `gap--group-to-group--vertical`? I'm not 100% those are the right tokens though.
  * There are no compact variants of gap tokens.
  * If we update to gap spacer tokens, should we also update the description list component?

File: src/patternfly/components/Banner/banner.scss [hunk 2]
```diff
@@ -20,9 +20,7 @@
   --#{$banner}--link--Color: var(--#{$banner}--Color);
```
Comment (mcoker) [inline src/patternfly/components/Banner/banner.scss:20]: If you hover over the "inline link button" link in the blue banner, the text color changes to blue.

I think you can just make these changes:
* Remove this `:hover` block for inline buttons - we should be able to handle this with the next point, and just set button vars. https://github.com/patternfly/patternfly/blob/728f578a7f74398df41f6920e1e4ae28976bf0b8/src/patternfly/components/Banner/banner.scss#L167-L169
* On this line, change `--#{$button}--m-link--m-inline--hover--Color` to `--#{$button}--m-link--hover--Color`. The previous var no longer exists, and this should set the button's hover color - https://github.com/patternfly/patternfly/blob/728f578a7f74398df41f6920e1e4ae28976bf0b8/src/patternfly/components/Banner/banner.scss#L162

File: src/patternfly/components/Timestamp/timestamp.scss [hunk 3]
```diff
@@ -2,13 +2,14 @@
 
 :where(:root, .#{$timestamp}) {
```
Comment (mcoker) [inline src/patternfly/components/Timestamp/timestamp.scss:3]: Super nit, would you mind adding a new timestamp to the existing list (or whatever you think is best) that shows the help text styling? Doesn't look like we have an example of that anywhere in the core docs.

You can get it by adding `timestamp--IsHelp=true` to the outer `{{#> timestamp}}` so it's `{{#> timestamp timestamp--IsHelp=true}}`

**Review by mcoker — COMMENTED** [review]
Looks good! Just a wee bit of feedback, nothing is blocking though if you'd prefer merge this as-is and follow up. Just lemme know if you'd prefer do that.

**Review by mcoker — COMMENTED** [review]
LGTM! Just want to confirm that we don't want to update the column/row gap for the content component `<dl>`? IMO it looks like there is too much space between term and description. Here's what it is currently:

Screenshot 2024-08-19 at 6 54 41 PM

And here's the [horizontal description list component](https://patternfly-pr-6953.surge.sh/components/description-list/#horizontal)'s column/row gap:

Screenshot 2024-08-19 at 6 54 59 PM
Screenshot 2024-08-19 at 6 54 53 PM

**Review by mcoker — APPROVED** [review]
LPTM! Backstop report if you're interested. [zandrew-the-true-king-of-the-north.psd.v2-final.ammend.client-approved.pdf](https://github.com/user-attachments/files/16716845/zandrew-the-true-king-of-the-north.psd.v2-final.ammend.client-approved.pdf)

Some notes about the report:

* Skip to content diff is just spacing from the content component `<dl>` on that example
* Label group diff - font (not)awesome didn't load
* Card with the missing kebab - icon font didn't load
* Wizard example with missing reference file - I think that's my oopsie from a previous visual regression report

**Review by srambach — APPROVED** [review]
✨ looks great!

**Pull request merged**